src/deps.test.ts

```diff
@@ -7,3 +7,19 @@ export {
 } from "https://deno.land/std@0.171.0/testing/asserts.ts";
 export { writableStreamFromWriter } from "https://deno.land/std@0.171.0/streams/writable_stream_from_writer.ts";
 export { serve } from "https://deno.land/std@0.171.0/http/server.ts";
+
+export async function withTempDir(action: (path: string) => Promise<void>) {
```

Comment:
I think this deserves a doc comment saying that it uses chdir internally, just because there is different ways to implement this and you can't guess from the name

Reply:
Done.